pegasus/sites.v3/code.org/views/tutorial_promo.haml

```diff
@@ -14,11 +14,12 @@
       %a{href: tutorial[:left_button_link], target: '_self'}
         %button.tutorial-button= tutorial[:left_button_text]
 
-  .col-25.specific-tutorial.right-tutorial
-    .tutorial-box{id: tutorial[:id_for_mobile_image]}
-      %h2.specific-tutorial-heading= tutorial[:right_heading]
-      .tutorial-details
-        %p.tutorial-description= tutorial[:right_description]
-        %p.tutorial-specs= tutorial[:right_specs]
-      %a{href: tutorial[:right_button_link], target: '_self'}
-        %button.tutorial-button= tutorial[:right_button_text]
+  - unless tutorial[:skip_right_box]
```

Comment:
We might be able to remove this block if there's only one variation?

Reply:
we still the second one on /starwars, which I didn't want to break.